backend/scripts/seed_dev.py

```
@@ -10,12 +11,21 @@
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "core.settings.dev")
 django.setup()
 
-from business.models import BusinessProfile  # noqa: E402  pylint: disable=wrong-import-position
+from business.models import (
+    BusinessProfile,
+)  # noqa: E402  pylint: disable=wrong-import-position
```

Ruff (0.13.3)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)